scripts/install-deps.sh

```diff
@@ -32,8 +32,7 @@ sudo pip3 install --upgrade pip virtualenv
 cd "${scripts_dir}/.."
 virtualenv --system-site-packages -p python3 env
 env/bin/pip install -r requirements.txt
-echo "/home/pi/AIY-projects-python/src" > \
-  /home/pi/AIY-projects-python/env/lib/python3.5/site-packages/aiy.pth
+echo "$(pwd)/src" > ./env/lib/python3.5/site-packages/aiy.pth
```

Comment:
$PWD is not guaranteed to be the the repository directory. Could you change it to use "${scripts_dir}/.." instead?